litani

```diff
@@ -416,6 +421,12 @@ def non_negative_int(arg):
     return ret
 
 
+def _non_directory_path(arg):
+    if pathlib.Path(arg).exists() and pathlib.Path(arg).resolve().is_dir():
+        raise ValueError("--out-file flag expects a file and not a directory")
```

Comment:
please include `arg` in the error message

Reply:
This is generated when we pass a dir - 
```
litani % ./litani dump-run --out-file /tmp/

usage: litani dump-run [-h] [-r N] [-o F]
litani dump-run: error: argument -o/--out-file: invalid _non_directory_path value: '/tmp/'
```